languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/client/files_producers/ClientConstants.kt

```diff
@@ -14,10 +15,14 @@ class ClientConstants @Inject constructor(
         val basePackageName: String
 ) {
 
-    val requestUtilsPackage = "${sharedPackage}/utils/requests-utils"
-    val uriUtilsPackage = "${sharedPackage}/utils/uri-utils"
-    val commonTypesPackage = "${sharedPackage}/utils/common-types"
-    val middlewarePackage = "${sharedPackage}/utils/middleware"
-    val apiRoot = "$clientPackage/api-root"
+    val requestUtilsPackage = "${sharedPackage.lowerCasePackage()}/utils/requests-utils"
```

Comment:
I don't think that this will break our code, but why was this change necessary? I thought that the user provides the `sharedPackage` when he invokes the generator? Then he could fix it in his invaocation? Or am I missing something here?

Reply:
actually there are packages that don't pass through the `VrapType` provider. In those cases you end up with two package naming, one lower case and one not. this is why i decided to unify it here ⬆️